src/main/java/teammates/ui/output/FeedbackSessionData.java

```diff
@@ -111,6 +116,20 @@ public FeedbackSessionData(FeedbackSessionAttributes feedbackSessionAttributes)
         } else {
             this.deletedAtTimestamp = feedbackSessionAttributes.getDeletedTime().toEpochMilli();
         }
+
+        this.studentDeadlines = feedbackSessionAttributes.getStudentDeadlines()
+                .entrySet()
+                .stream()
+                .collect(Collectors.toMap(Map.Entry::getKey, entry ->
+                        TimeHelper.getMidnightAdjustedInstantBasedOnZone(entry.getValue(), timeZone, true)
+                                .toEpochMilli()));
```

Comment:
```suggestion
        this.studentDeadlines = feedbackSessionAttributes.getStudentDeadlines()
                .entrySet()
                .stream()
                .collect(Collectors.toMap(
                        email -> email, 
                        deadline -> TimeHelper.getMidnightAdjustedInstantBasedOnZone(deadline, timeZone, true)
                                .toEpochMilli()));
```
Something like this?

Reply:
Unfortunately, this does not work. What is streamed are instances of `Map.Entry` so `email` and `deadline` variables would actually both have the type of `Map.Entry`, not `String` and `Instant` respectively.
![image](https://user-images.githubusercontent.com/65202977/157932401-1964dab4-ab23-4ed8-81a1-e20d2257e37f.png)
This is why I am doing that, even though it is kind of ugly and I wish I could do what you suggested lol

Reply:
Alright... seems like it:(